app/Website/src/Http/Controllers/NewsletterSubscribeController.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace App\Website\Http\Controllers;
+
+use App\Http\Controllers\Controller;
+use Knuckles\Scribe\Attributes\Endpoint;
+use Spatie\MailcoachSdk\Facades\Mailcoach;
+use App\Website\Http\Requests\SubscribeRequest;
+
+class NewsletterSubscribeController extends Controller
+{
+    #[Endpoint('Website/Subscribe', <<<'DESC'
+  This endpoint is for subscribing to Leanhub.mk.
+  Additionally it uses Cloudflare Turnstile ReCaptcha for validation.
+ DESC)]
+    public function store(SubscribeRequest $request)
```

Comment:
Please add a group attribute here with the name of `Website` to group the endpoint in the docs under `Website`.

Reply:
Done